R/utils.R

```diff
@@ -145,12 +145,16 @@ quote_wrap <- function(x, q) paste0(q, x, q)
 # interface to work like options() or setwd() -- returns the old value for convenience
 set_lang <- function(new_lang) {
   old_lang <- Sys.getenv("LANGUAGE", unset = NA)
-  Sys.setenv(LANGUAGE = new_lang)
+  Sys.setenv(LANGUAGE = new_lang) # nolint: undesirable_function_linter. Avoiding {withr} dep in pkg.
```

Comment:
We usually skip the `_linter` suffix for shorter comments.

Reply:
Ah, made me realize #2086